src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -344,34 +344,34 @@ static void WindowLoadsaveResize(rct_window* w)
 static u8string Browse(bool isSave)
 {
     OpenRCT2::Ui::FileDialogDesc desc = {};
-    u8string extension = "";
+    u8string extension = u8"";
```

Comment:
I would rather not have it do that, this will copy an empty string, it may or may not be optimized by the compiler due to dead stores but debug builds never see the light of such things, default initialized strings are empty.